lib/modules/datasource/cdnjs/index.ts

```diff
@@ -1,7 +1,30 @@
+import { ZodError, z } from 'zod';
+import { logger } from '../../../logger';
 import { ExternalHostError } from '../../../types/errors/external-host-error';
+import type { HttpError } from '../../../util/http';
+import { Result } from '../../../util/result';
 import { Datasource } from '../datasource';
-import type { GetReleasesConfig, ReleaseResult } from '../types';
-import type { CdnjsResponse } from './types';
+import { ReleasesConfig } from '../schema';
+import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+
+const Homepage = z.string().optional().catch(undefined);
+
+const Repository = z
+  .object({
+    type: z.literal('git'),
+    url: z.string(),
+  })
+  .transform(({ url }) => url)
+  .optional()
+  .catch(undefined);
+
+const Assets = z.array(
+  z.object({
+    version: z.string(),
+    files: z.string().array(),
+    sri: z.record(z.string()).optional(),
+  })
+);
```

Comment:
why not moving those to schema.ts too?

Reply:
The code for this datasource is not too big, so I think it's better to contain all of it in the single file

Reply:
then also inline the one zod schema which resides in schema.ts

Reply:
But there is no `schema.ts`

Reply:
🙈 sorry, I misread the datasource general schema.ts 😅